apps/console/src/auth.html

```diff
@@ -207,83 +253,151 @@
                             : "";
+                    };
+
+                    /**
+                     * Construct the sign-in redirect URL.
+                     *
+                     * @returns {string} Contructed URL.
+                     */
+                    function signInRedirectURL() {
+                        if (getTenantName() === startupConfig.superTenant) {
+                            return applicationDomain.replace(/\/+$/, '')
+                                + "<%= htmlWebpackPlugin.options.basename ? '/' + htmlWebpackPlugin.options.basename : ''%>";
+                        }
+
+                        return applicationDomain.replace(/\/+$/, '') + getTenantPath()
+                            + "<%= htmlWebpackPlugin.options.basename ? '/' + htmlWebpackPlugin.options.basename : ''%>";
                     }
 
-                    var auth = AsgardeoAuth.AsgardeoSPAClient.getInstance();
+                    /**
+                     * Construct the sign-out redirect URL.
+                     *
+                     * @returns {string} Contructed URL.
+                     */
+                    function getSignOutRedirectURL() {
+                        if (getTenantName() === startupConfig.superTenant) {
+                            return applicationDomain.replace(/\/+$/, '');
+                        }
+
+                        return applicationDomain.replace(/\/+$/, '') + getTenantPath();
+                    }
+
+                    /**
+                     * Construct the auth params for organization login `authorize` requets.
+                     *
+                     * @remarks This only applies to the new authz runtime.
+                     *
+                     * @returns {string} Contructed auth params.
+                     */
+                    function getAuthParamsForOrganizationLogins(orginalParams) {
+                        if (startupConfig.legacyAuthzRuntime) {
+                            return orginalParams;
+                        }
+
+                        var authParams = Object.assign({}, orginalParams);
+
+                        if (getOrganizationPath()) {
+                            var initialUserOrgInLocalStorage = localStorage.getItem("user-org");
+                            var orgIdInLocalStorage = localStorage.getItem("org-id");
+
+                            if (orgIdInLocalStorage) {
+                                if (orgIdInLocalStorage === getOrganizationName() && initialUserOrgInLocalStorage !== "undefined") {
+                                    authParams["fidp"] = "OrganizationSSO";
+                                    authParams["orgId"] = getOrganizationName();
+                                }
+                            } else {
+                                authParams["fidp"] = "OrganizationSSO";
+                                authParams["orgId"] = getOrganizationName();
+                            }
```

Comment:
This if else block seems to be redundant. Shall we verify whether these are expected?